cli/src/main/java/org/jboss/as/cli/handlers/LsHandler.java

```diff
@@ -571,6 +571,9 @@ private ModelNode getAddressNode(CommandContext ctx, OperationRequestAddress add
             while (iterator.hasNext()) {
                 OperationRequestAddress.Node node = iterator.next();
                 if (node.getName() != null) {
+                    if (node.getName().equals("*")) {
+                        throw new CommandFormatException("* is not supported in node path argument");
```

Comment:
@jfdenise Do we use the 'node' much in CLI docs, help, etc?

I think I've seen it before but it's a bit odd to use that term  in messages unless it's the typical way navigation is described.